Makefile

```diff
@@ -101,32 +110,55 @@ gcloud-test-cluster: NETWORK ?= default
 gcloud-test-cluster:
 	./scripts/create-cluster.sh ${GCP_CLUSTER_NODEPOOL_INITIALNODECOUNT} ${GCP_CLUSTER_NODEPOOL_MACHINETYPE} ${LOCATION} ${NETWORK}
 
+.PHONY: helm-repo-add
+helm-repo-add: 
+	helm repo add $(AGONES_NS) https://agones.dev/chart/stable
+	helm repo add $(OM_NS) https://open-match.dev/chart/stable
+	helm repo update
+
+.PHONY: helm-repo-remove
+helm-repo-remove: 
+	helm repo remove $(AGONES_NS)
+	helm repo remove $(OM_NS) 
+
 # install agones in local-cluster
 .PHONY: agones-install-local
 agones-install-local:
-	make agones-install
-	kubectl scale deploy agones-allocator --replicas 1 -n $(AGONES_NS)
-	kubectl scale deploy agones-ping --replicas 0 -n $(AGONES_NS)
+	helm install $(AGONES_NS) --namespace $(AGONES_NS) \
+		--create-namespace $(AGONES_NS)/agones \
+		--version $(AGONES_VER) \
+		--set agones.ping.install=false \
+		--set agones.allocator.replicas="1"
 
 # install agones
 .PHONY: agones-install
 agones-install:
-	kubectl create namespace $(AGONES_NS)
-	kubectl apply --server-side -f https://raw.githubusercontent.com/googleforgames/agones/release-$(AGONES_VER)/install/yaml/install.yaml
+	helm install ${AGONES_NS} --namespace ${AGONES_NS} \
+		--create-namespace $(AGONES_NS)/agones \
+		--version ${AGONES_VER}
 
 # uninstall agones and agones resources
 .PHONY: agones-uninstall
 agones-uninstall:
 	kubectl delete fleets --all --all-namespaces
 	kubectl delete gameservers --all --all-namespaces
-	kubectl delete -f https://raw.githubusercontent.com/googleforgames/agones/release-$(AGONES_VER)/install/yaml/install.yaml
+	helm uninstall $(AGONES_NS) --namespace $(AGONES_NS)
+	kubectl delete namespace $(AGONES_NS)
+
+# uninstall agones and agones resources in local-cluster
+.PHONY: agones-uninstall-local
+agones-uninstall-local:
+	kubectl delete fleets --all --all-namespaces
+	kubectl delete gameservers --all --all-namespaces
```

Comment:
Agnoes helm charat has a pre-hook to delete agones resources when chart uninstalling.
So you don't have to delete these.
https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/hooks/pre_delete_hook.yaml

Reply:
Thanks for reviewing. I deleted `kubectl` commands for `agones-uninstall` and `agones-uninstall-local`

Reply:
LGTM